src/System.Net.Http/tests/FunctionalTests/DiagnosticsTests.cs

```diff
@@ -258,6 +258,7 @@ public void SendAsync_ExpectedDiagnosticExceptionLogging()
             }).Dispose();
         }
 
+        [ActiveIssue(23209)]
```

Comment:
Disabling it for now is fine.  Seems like a minimal mitigation would be to a) simply increase the timeout to something much larger than 1 second, and b) stop the spinning that's being done and instead set an event or a tcs or something that can be blocked on rather than just doing volatile read/writes with a Boolean.

Reply:
I don't it's failing because it's taking too long to cancel.  I think it is failing because the network operation happens so fast and completes before it can be canceled.